src/site/content/en/learn/css/the-cascade/index.md

```diff
@@ -0,0 +1,278 @@
+By making a rule more specific,
+you can cause it to be applied even if some other CSS that matches the selector appears later in the CSS.
+
+In the next lesson you can learn the details of how specificity is calculated,
+however keeping a few things in mind will help you avoid too many specificity issues.
+
+CSS targeting a class on an element will make that rule more specific,
+and therefore seen as more important to be applied,
+than CSS targeting the element alone.
+This means that with the following CSS,
+the `h1` will be colored red even though both rules match and the rule for the `h1` selector comes later in the stylesheet.
+
+```html
+<h1 class="my-element">Heading</h1>
+```
+
+```css
+.my-element {
+  color: red;
+}
+
+h1 {
+  color: blue;
+}
+```
+
+An `id` makes the CSS even more specific,
+so styles applied to an ID will override those applied many other ways.
+This is one reason why it is generally not a good idea to attach styles to an `id`.
+It can make it difficult to overwrite that style with something else.
+
+### Specificity is cumulative
+
+As you can find out in the next lesson,
+each type of selector is awarded points which indicate how specific it is,
+the points for all of the selectors you have used to target an element are added together.
+This means that if you target an element with a selector list such as
+`a.my-class.another-class[href]:hover` you get something quite hard to overwrite with other CSS.
+For this reason, and to help make your CSS more reusable,
+it's a good idea to keep your selectors as simple as possible.
+Use specificity as a tool to get at elements when you need to,
+but always consider refactoring long, specific selector lists, if you can.
+
+## Origin
+
+The CSS that you write isn't the only CSS applied to a page.
+The cascade takes into account the origin of the CSS.
+This origin includes the browser's internal stylesheet,
+styles added by browser extensions or the operating system,
+and your authored CSS.
+The **order of specificity of these origins**, from least specific, to most specific is as follows:
+
+1. **User agent base styles**. These are the styles that your browser applies to HTML elements by default.
+2. **Local user styles**. These can come from the operating system level, such as a base font size,
+or a preference of reduced motion.
+They can also come from browser extensions,
+such as a browser extension that allows a user to write their own custom CSS for a webpage.
+3. **Authored CSS**. The CSS that you author.
+4. **Authored `!important`**. Any `!important` that you add to your authored declarations.
+5. **Local user styles `!important`**. Any `!important` that come from the operating system level,
+or browser extension level CSS.
+6. **User agent `!important`**. Any `!important` that are defined in the default CSS,
+provided by the browser.
+
+<figure class="w-figure">
+{% Img src="image/VbAJIREinuYvovrBzzvEyZOpw5w1/rjZMVAC5lNATzCErbyeI.png",
```

Comment:
Do we have an SVG of this?

Reply:
Yeh absolutely. I'm just putting bitmaps up for diagrams while we get things ready to go, so I don't have to spend time messing around converting text elements to paths etc to much. We should add switching these things to SVG as a pre-flight check.